dpnp/linalg/dpnp_iface_linalg.py

```diff
@@ -901,6 +903,68 @@ def lstsq(a, b, rcond=None):
     return dpnp_lstsq(a, b, rcond=rcond)
 
 
+def lu_factor(a, overwrite_a=False, check_finite=True):
```

Comment:
seems CuPy puts these SciPy-style interfaces into a dedicated `scipy` namespace

should we consider the same?

Reply:
@ndgrigorian yes, CuPy has a separate namespace for `SciPy` functions. 
I thought about doing the same but since we do not have many functions, I decided to do it in the future

Reply:
@vlad-perevezentsev 
fair point, sounds good to me